apps/src/templates/projects/ProjectCard.jsx

```diff
@@ -0,0 +1,250 @@
+  },
+  thumbnail: {
+    width: 215,
+    height: 150
+  },
+  actionBox: {
+    width: 160,
+    paddingLeft: 15,
+    paddingRight: 15,
+    border: '1px solid #bbbbbb',
+    borderRadius: 2,
+    backgroundColor: color.white,
+    boxShadow: "3px 3px 3px gray",
+    marginTop: 5,
+    position: "absolute"
+  },
+  actionText: {
+    fontSize: 12,
+    fontFamily: '"Gotham", sans-serif',
+    color: color.gray
+  },
+  delete: {
+    color: color.red,
+    borderTop: '1px solid lightGray',
+    paddingTop: 10,
+    fontSize: 12
+  },
+  xIcon: {
+    paddingRight: 5
+  },
+};
+
+const ProjectCard = React.createClass({
+  propTypes: {
+    projectData: React.PropTypes,
+    currentGallery: React.PropTypes.string,
+  },
+
+  getInitialState() {
+   return {actionsOpen: false};
+  },
+
+  dateFormatter(dateString) {
+    const date = new Date(dateString);
+    return date.toLocaleDateString();
+  },
+
+  timeFormatter(dateString) {
+    const date = new Date(dateString);
+    return date.toLocaleTimeString();
+  },
+
+  renderStudentName() {
+    // The student's name should only be visible in the classroom gallery.
+    if (this.props.currentGallery === 'class'){
+      return (
+        <div style={styles.studentName}>
+          By: {this.props.projectData.studentName}
+        </div>
+      );
+    }
+  },
+
+  renderFirstInitial() {
+    if (this.props.currentGallery === 'public'){
+      return (
+        <span style={styles.firstInitial}>
+          By: {this.props.projectData.studentName[0]}
+        </span>
+      );
+    }
+  },
+
+  renderStudentAgeRange() {
+    // The student's age range should only be visible in the public gallery.
+    if (this.props.currentGallery === 'public'){
+      if (this.props.projectData.studentAge >= 18){
+        return (
+          <span style={styles.ageRange}>
+            Age: 18+
+          </span>
+        );
+      }
+      if (this.props.projectData.studentAge >= 13){
+        return (
+          <span style={styles.ageRange}>
+            Age: 13+
+          </span>
+        );
+      }
+      if (this.props.projectData.studentAge >= 8){
+        return (
+          <span style={styles.ageRange}>
+            Age: 8+
+          </span>
+        );
+      }
+      if (this.props.projectData.studentAge >= 4){
+        return (
+          <span style={styles.ageRange}>
+            Age: 4+
+          </span>
+        );
+      }
+    }
+  },
+
+  renderArrowIcon() {
+    // Only the student can access the menu options when viewing their personal projects.
+   if (this.props.currentGallery === 'personal' && !this.state.actionsOpen){
+     return (
+       <FontAwesome icon=" fa-chevron-down" style={styles.arrowIcon} onClick={this.toggleActionBox}/>
+      );
+    }
+    if (this.props.currentGallery === 'personal' && this.state.actionsOpen){
+      return (
+        <FontAwesome icon=" fa-chevron-up" style={styles.arrowIcon} onClick={this.toggleActionBox}/>
+       );
+    }
+  },
```

Comment:
I think FontAwesome expects just `chevron-up/down` as input rather than fa-*.

Also, you could avoid some repetition by doing something like `icon = this.state.actionsOpen ? 'chevron-up' : 'chevron-down'` then later `<FontAwesome icon={icon} .../>`